desktop/core/src/desktop/js/reactComponents/FileChooser/PathBrowser/PathBrowser.tsx

```diff
@@ -94,11 +107,14 @@ const PathBrowser: React.FC<PathBrowserProps> = ({ breadcrumbs, handleFilePathCh
     return (
       <>
         {!isEditMode ? (
-          <div className="hue-path-browser">
-            <div className="hue-filesystem__icon">
+          <div className="hue-path-browser" data-testid={`${testId}`}>
+            <div
```

Comment:
You could simply avoid rendering the icon if it should not be visible. Then you have a cleaner DOM and no need for additional styling.